articles/azure-monitor/app/monitor-performance-live-website-now.md

```diff
@@ -143,6 +143,8 @@ We are tracking this issue [here](https://github.com/Microsoft/ApplicationInsigh
 * To output verbose logs, modify the config file: `C:\Program Files\Microsoft Application Insights\Status Monitor\Microsoft.Diagnostics.Agent.StatusMonitor.exe.config` and add `<add key="TraceLevel" value="All" />` to the `appsettings`.
 Then restart status monitor.
 
+* As Status Monitor is a .net application you can also enable [.net tracing by adding the appropriate diagnostics to the config file](https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/trace-debug/system-diagnostics-element). For example, in some scenarios it can be useful to see what's happening at the network level by [configuring network tracing](https://docs.microsoft.com/en-us/dotnet/framework/network-programming/how-to-configure-network-tracing)
```

Comment:
```suggestion
* As Status Monitor is a .NET application you can also enable [.net tracing by adding the appropriate diagnostics to the config file](https://docs.microsoft.com/dotnet/framework/configure-apps/file-schema/trace-debug/system-diagnostics-element). For example, in some scenarios it can be useful to see what's happening at the network level by [configuring network tracing](https://docs.microsoft.com/dotnet/framework/network-programming/how-to-configure-network-tracing)
```

Reply:
When en-us is present in a link it hardcodes the link to English, versus when it is not present docs.microsoft.com will attempt to localize the returned page for each customer who clicks the link,